### PR TITLE
Added HTML for section 3

### PR DESCRIPTION
**Pull request by Denis20101977**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,7 +43,18 @@ <h2 class="warranty__title" warranty boost</h2>
         </div>
       </div>
     <!-- Секция 3 -->
-    <section>Секция 3</section>
+    <section class="care" id="care">
+      <div class="care__wrapper">
+        <div class="care__text-block">
+          <h2 class="care__title" ZEN care</h2>
```
Comment (Palych18) [inline index.html:49]: Angle bracket

File: index.html [hunk 2]
```diff
@@ -43,7 +43,18 @@ <h2 class="warranty__title" warranty boost</h2>
...
+          <p class="care__copy">Shopping online is usually a delightful experience. Until it's not.</p>
+          <p class="care__copy">Until something you buy, proves to be faulty, missing or a totally wrong item, youre in for long product return procedures.</p> 
+          <p class="care__copy">But, here comes ZEN Care: a free, built-in shopping protection.</p>  
+        </div>
+        <div class="care__image-block">
+          <img src="./assets/images/3.jpg" alt="Cards" />
+        </div>
+      </div>
```
Comment (Palych18) [inline index.html:57]: Close tag

**Review by Palych18 — COMMENTED** [review]
Fix

File: index.html (hunk 1 above)
Comment (Denis20101977) [reply index.html]: fixed.

File: index.html (hunk 2 above)
Comment (Denis20101977) [reply index.html:58]: fixed.

**Review by Palych18 — APPROVED** [review]
Done

**Pull request merged**